src/cockpit/389-console/src/lib/database/globalPwp.jsx

```diff
@@ -1427,6 +1515,97 @@ export class GlobalPwPolicy extends React.Component {
                                 {saveBtnName}
                             </Button>
                         </Tab>
+                        <Tab eventKey={4} title={<TabTitleText>Temporary Password Rules</TabTitleText>}>
+                            <Form className="ds-margin-top-xlg ds-margin-left" isHorizontal autoComplete="off">
+                                {this.state.passwordmustchange == false && (
+                                <FormAlert>
+                                    <Alert
+                                    variant="info"
+                                    title='"User Must Change Password After Reset" must be enabled in General Settings to avail of TPR.'
```

Comment:
Sorry grammar seems funny here:

User Must Change Password After Reset" must be enabled in General Settings to avail of TPR

Perhaps change "avail" to "active"?

Reply:
How about this:

"User Must Change Password After Reset" must be enabled in General Settings to activate TPR

Reply:
Or "enable" instead of "activate", but "avail" is not a term I understand in this context.

Reply:
LGMT thanks!